packages/bootstrap/extra/dcos_internal_utils/exhibitor.py

```diff
@@ -35,6 +39,9 @@ def try_shortcut():
     # Make sure that the pid hasn't been written anew
     cur_pid_stat = get_zk_pid_mtime()
 
+    if cur_pid_stat is None:
+        return False
+
```

Comment:
Where this function is called it previously might have raised FileNotFoundError, and now `False` is returned.

In this patch, I however do not see a consumer changed that handles the case where this function returns `False`.

Reply:
Please check the line 75, it is outside of the diff.

The FileNotFound exception being unhandled is the main reason why the bootstrap script was failing. The `try_shortcut()` function should be returning only `True/False`.

Reply:
I see below we do 
```
     zk_pid = get_zk_pid()
     cmdline_path = '/proc/{}/cmdline'.format(zk_pid)
```

Can it now happen that we construct a `cmdline_path` with the value `/proc/False/cmdline`?

Reply:
Oh wow now I get that we did not modify the function `get_zk_pid()`.

When I wrote

> Where this function is called it previously might have raised 

I assumed that we modify `get_zk_pid()` (as shown by the GitHub diff) and not `try_shortcut()` (hidden by the GitHub diff).